### PR TITLE
API-7063 v3.3: add list_routing_statuses to agent-api.

### PR DESCRIPTION
**Pull request by gluszczyk**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/API-7063-list-routing-statuses)
- [Jira](https://livechatinc.atlassian.net/browse/API-7063)

### Content

- New content

### CODE REVIEW

File: content/messaging/agent-chat-api/index.mdx [hunk 1]
```diff
@@ -2739,6 +2739,64 @@ https://api.livechatinc.com/v3.2/agent/action/list_agents_for_transfer \
...
+### List Routing Statuses
+
+Returns current routing status of every agent selected by the provided filters.
+
+#### Specifics
+
+|                        |                                                                          |
+| ---------------------- | ------------------------------------------------------------------------ |
+| **Method URL**         | `https://api.livechatinc.com/v3.2/agent/action/list_routing_statuses` |
+| **Required scopes**    | -                                                                        |
```
Comment (kogierman) [inline content/messaging/agent-chat-api/index.mdx:2754]: 360noscope? :trollface:

File: content/messaging/agent-chat-api/rtm-reference/index.mdx [hunk 2]
```diff
@@ -3412,6 +3412,70 @@ It returns the Agents you can transfer a chat to. Agents are sorted ascendingly
...
+### List Routing Statuses
+
+Returns current routing status of every agent selected by the provided filters.
+
+#### Specifics
+
+|                        |                                                      |
+| ---------------------- | ---------------------------------------------------- |
+| **Action**             | `list_routing_statuses`                              |
+| **Required scopes**    | -                                                    |
```
Comment (kogierman) [inline content/messaging/agent-chat-api/rtm-reference/index.mdx:3427]: same as web

**Review by kogierman — CHANGES_REQUESTED** [review]

File: content/messaging/agent-chat-api/index.mdx (hunk 1 above)
Comment (kogierman) [reply content/messaging/agent-chat-api/index.mdx:2754]: also - v3.2?

Comment (gluszczyk) [reply content/messaging/agent-chat-api/index.mdx:2754]: ehh, too much copy-pasta *burp*

File: content/messaging/agent-chat-api/changelog/index.mdx [hunk 3]
```diff
@@ -56,6 +56,9 @@ This document is the record of all the changes in the **Agent Chat API**, Web an
 
 - The **Login** ([RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#login)) method has a new optional parameter, `customer_push_level`. It controls the volume of customer-related pushes sent to the agent.
 
+### Other
+- There's a new method **List Routing Statuses** ([Web](/messaging/agent-chat-api/v3.3/#list-routing-statuses) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#list-routing-statuses)) for retrieving current routing status of every agent.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/changelog/index.mdx:60]:
```suggestion
- There's a new method, **List Routing Statuses** ([Web](/messaging/agent-chat-api/v3.3/#list-routing-statuses) & [RTM](/messaging/agent-chat-api/v3.3/rtm-reference/#list-routing-statuses)), for retrieving the agent's current routing status.
```

File: content/messaging/agent-chat-api/index.mdx [hunk 4]
```diff
@@ -2739,6 +2739,64 @@ https://api.livechatinc.com/v3.2/agent/action/list_agents_for_transfer \
 </Code>
 </Section>
 
+<Section>
+<Text>
+
+### List Routing Statuses
+
+Returns current routing status of every agent selected by the provided filters.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/index.mdx:2747]:
```suggestion
Returns the current routing status of each agent selected by the provided filters.
```

File: content/messaging/agent-chat-api/index.mdx [hunk 5]
```diff
@@ -2739,6 +2739,64 @@ https://api.livechatinc.com/v3.2/agent/action/list_agents_for_transfer \
...
+| `filters`           | No       | `object` |                                         |
+| `filters.group_ids` | No       | `array`  | Array of group IDs. Max array size: 200 |
+
+</Text>
+<Code>
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+https://api.livechatinc.com/v3.2/agent/action/list_routing_statuses \
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/index.mdx:2771]:
```suggestion
https://api.livechatinc.com/v3.3/agent/action/list_routing_statuses \
```

File: content/messaging/agent-chat-api/rtm-reference/index.mdx [hunk 6]
```diff
@@ -3412,6 +3412,70 @@ It returns the Agents you can transfer a chat to. Agents are sorted ascendingly
 </Code>
 </Section>
 
+<Section>
+<Text>
+
+### List Routing Statuses
+
+Returns current routing status of every agent selected by the provided filters.
```
Comment (ale-kacperczyk) [inline content/messaging/agent-chat-api/rtm-reference/index.mdx:3420]:
```suggestion
Returns the current routing status of each agent selected by the provided filters.
```

**Review by ale-kacperczyk — CHANGES_REQUESTED** [review]

**Review by kogierman — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**